### PR TITLE
Added match status

### PR DESCRIPTION
**Pull request by Chuli2k**
A match can have different statuses depending on what happened: live, match over, sheduled(countdown timer is displayed) and match postponed. 
I added an enum with all the statuses and a property to the FullMatch interface.

This is helpful if you want to know if the match is still going to happen or is actually over. For example if there was a fortfeit, there is no result, the match is past it's date, but is still over.

I left the `live` property for backwards compatibility.

Examples:
- Postponed: [https://www.hltv.org/matches/2333181/-](https://www.hltv.org/matches/2333181/-)
- Over, due to withdraw of one team: [https://www.hltv.org/matches/2332985/-](https://www.hltv.org/matches/2332985/-)

### CODE REVIEW

File: src/playground.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import HLTV from './index'
 //     console.dir(data, { depth: null })
 // }})
 // HLTV.getMatchesStats({startDate: '2017-07-10', endDate: '2017-07-18'}).then(res => console.log(res.length))
-// HLTV.getMatch({id: 2312432}).then(res => console.dir(res, {depth: null})).catch(err => console.log(err))
+HLTV.getMatch({id: 2332985}).then(res => console.dir(res, {depth: null})).catch(err => console.log(err))
```
Comment (gigobyte) [inline src/playground.ts]: Can you please discard the changes in this file?

File: src/enums/MatchStatus.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export enum MatchStatus {
+  live = 'LIVE',
```
Comment (gigobyte) [inline src/enums/MatchStatus.ts]: Enums in TypeScript are capitalized by convention.

**Comment by gigobyte** [comment]
I left some comments, other than that - looks good! Also can you please run `yarn test -u` so that the snapshots get updated?

File: src/playground.ts (hunk 1 above)
Comment (Chuli2k) [reply src/playground.ts]: Done.

File: src/enums/MatchStatus.ts (hunk 2 above)
Comment (Chuli2k) [reply src/enums/MatchStatus.ts]: Done.

**Comment by Chuli2k** [comment]
Made the changes and updated the snapshots.

**Comment by gigobyte** [comment]
Thanks, I released a new 2.14.3 version of `hltv`

**Pull request merged**